applications/fishing-map/src/features/datasets/NewDataset.tsx

```diff
@@ -0,0 +1,226 @@
+import React, { useState, useCallback } from 'react'
+import cx from 'classnames'
+import { useDropzone } from 'react-dropzone'
+import { useTranslation } from 'react-i18next'
+import InputText from '@globalfishingwatch/ui-components/dist/input-text'
+import Modal from '@globalfishingwatch/ui-components/dist/modal'
+import Button from '@globalfishingwatch/ui-components/dist/button'
+// import Select from '@globalfishingwatch/ui-components/dist/select'
+import { ReactComponent as FilesIcon } from 'assets/icons/files-supported.svg'
+import { USER_CONTEXT_TYPE } from 'data/datasets'
+import { useDatasetsAPI, useDatasetModalConnect } from './datasets.hook'
+import styles from './NewDataset.module.css'
+
+interface DatasetConfigProps {
+  className?: string
+  onDatasetFieldChange: (field: any) => void
+  metadata: {
+    name: string
+  }
+}
+
+const DatasetConfig: React.FC<DatasetConfigProps> = (props) => {
+  const { metadata, className = '', onDatasetFieldChange } = props
+  const { t } = useTranslation()
+  return (
+    <div className={cx(styles.datasetConfig, className)}>
+      <InputText
+        inputSize="small"
+        value={metadata.name}
+        label={t('common.name', 'Name')}
+        className={styles.input}
+        onChange={(e) => onDatasetFieldChange({ name: e.target.value })}
+      />
+      <InputText
+        inputSize="small"
+        label={t('common.description', 'Description')}
+        className={styles.input}
+        onChange={(e) => onDatasetFieldChange({ description: e.target.value })}
+      />
+      {/*
```

Comment:
I'd normally remove the commented code

Reply:
Indeed, but in this case I'm going to need it in the next PR